package.json

```diff
@@ -25,7 +25,8 @@
     "test:ci": "npm test -- --ci --maxWorkers=2  --reporters=default --reporters=jest-junit",
     "codegen": "graphql-codegen --config codegen.yml",
     "codegen:check": "npm run codegen && git diff --exit-code",
-    "lint": "eslint . --ext .ts"
+    "lint": "eslint . --ext .ts",
+    "gen-error-code-doc": "node ./internals-js/dist/genErrorCodeDoc.js > ./docs/source/errors.md"
```

Comment:
Can we enforce the error doc is updated in CI similar to the above `codegen:check`?
The related Circle job is here: https://github.com/apollographql/federation/blob/main/.circleci/config.yml#L84-L94

Reply:
Yes, that make sense and I did that. I am a circleci n00b however so appreciate if you can double-check https://github.com/apollographql/federation/pull/1274/commits/8e14b8cc565d06e6fd28f865a514053dec97671a (not too worried cause I genuinely just copied what `codegen:check` but ...).